### ISSUE TITLE
Is share_textvqa is the same to textvqa in llava dataset? 

### ISSUE THREAD

**Issue by cooleel** [TinyLLaVA/TinyLLaVA_Factory#65]
https://tinyllava-factory.readthedocs.io/en/latest/Prepare%20Datasets.html#organize-data
from the description here : 
>TextVQA: Already download as “TextVQA” in LLaVA’s SFT Images.

<img width="679" alt="Screenshot 2024-05-24 at 11 14 43 AM" src="https://github.com/TinyLLaVA/TinyLLaVA_Factory/assets/10917451/f0178027-9ca7-494b-b740-265f17d26886">

**Comment by YingHuTsing** [comment]
No. they are not the same. share_textvqa can be get from here https://drive.google.com/drive/folders/1tCUQ-sq6vdshZVkF0ZeF3K4eztkXJgax according to our Doc.
![1716633076212](https://github.com/TinyLLaVA/TinyLLaVA_Factory/assets/168290753/fa966f92-c528-4aa4-99b0-ad284e7ab9fd)
